ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -100,3 +82,186 @@ iroha_docker_logging_max_file: 3
+                    "can_get_all_signatories",
+                    "can_get_all_txs",
+                    "can_get_blocks",
+                    "can_get_roles",
+                    "can_read_assets",
+                    "can_remove_signatory",
+                    "can_set_quorum"
+                  ],
+                  "roleName":"admin"
+                }
+              },
+              {
+                "createRole":{
+                  "permissions":[
+                    "can_add_signatory",
+                    "can_get_my_acc_ast",
+                    "can_get_my_acc_ast_txs",
+                    "can_get_my_acc_detail",
+                    "can_get_my_acc_txs",
+                    "can_get_my_account",
+                    "can_get_my_signatories",
+                    "can_get_my_txs",
+                    "can_grant_can_add_my_signatory",
+                    "can_grant_can_remove_my_signatory",
+                    "can_grant_can_set_my_account_detail",
+                    "can_grant_can_set_my_quorum",
+                    "can_grant_can_transfer_my_assets",
+                    "can_receive",
+                    "can_remove_signatory",
+                    "can_set_quorum",
+                    "can_transfer"
+                  ],
+                  "roleName":"user"
+                }
+              },
+              {
+                "createRole":{
+                  "permissions":[
+                    "can_add_asset_qty",
+                    "can_create_asset",
+                    "can_receive",
+                    "can_transfer"
+                  ],
+                  "roleName":"money_creator"
+                }
+              },
+              {
+                "createDomain":{
+                  "defaultRole":"user",
+                  "domainId":"test"
+                }
+              },
+              {
+                "createAsset":{
+                  "assetName":"coin",
+                  "domainId":"test",
+                  "precision":2
+                }
+              },
+              {
+                "createAccount":{
+                  "accountName":"admin",
+                  "domainId":"test",
+                  "publicKey":"313a07e6384776ed95447710d15e59148473ccfc052a681317a72a69f2a49910"
+                }
+              },
+              {
+                "createAccount":{
+                  "accountName":"test",
+                  "domainId":"test",
+                  "publicKey":"716fe505f69f18511a1b083915aa9ff73ef36e6688199f3959750db38b8f4bfc"
+                }
+              },
+              {
+                "appendRole":{
+                  "accountId":"admin@test",
+                  "roleName":"admin"
+                }
+              },
+              {
+                "appendRole":{
+                  "accountId":"admin@test",
+                  "roleName":"money_creator"
+                }
+              },
+              {
+                "addPeer":{
```

Comment:
delete all addPeer from genesis_block

Reply:
I thing that we haven't to change it, because it can help us to check correct work peer clear operation in the ansible role with default genesis_block.